pkg/controller/networkpolicy/clustergroup.go

```diff
@@ -165,8 +165,8 @@ func (n *NetworkPolicyController) enqueueInternalGroup(key string) {
 	n.internalGroupQueue.Add(key)
 }
 
-func (c *NetworkPolicyController) internalGroupWorker() {
-	for c.processNextInternalGroupWorkItem() {
+func (n *NetworkPolicyController) internalGroupWorker() {
```

Comment:
most of  receivers are using "n", that's why the warning suggests to update `c` to `n`, I don't think we can say `c` is correct or not correct since it's just a variable name, I suppose it's Ok to use `n`, any strong opinion to use `c`

Reply:
The object is controller, not networkpolicy, IMO c is more proper than n as the short name. My point is, since the PR is unifying the short name, it should unify to the more proper one. But maybe it's hard to get everyone on same page on which one is more proper.

Reply:
yeah, if you don't mind, I'd like to keep it as `n`, if we change it to `c` for other methods(too many than this two methods with `c` receiver), there are a quite a lot code changes in methods' body for this receiver name update.

Reply:
I'm fine with it. Will defer to @antoninbas to final review since he had other comments.

Reply:
@antoninbas  I suppose you actually want to say "using c for all methods"?

Reply:
yes, my bad, that's I meant `c`

Reply:
ok, I have replaced all `n` with `c`